org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/problem/messages.properties

```diff
@@ -905,6 +905,7 @@
 1106 = Preview features enabled at an invalid source release level {0}, preview can be enabled only at source level {1}
 1107 = The Java feature ''{0}'' is only available with source level {1} and above
 1108 = You are using an API that is part of a preview feature and may be removed in future
+1109 = The Java version ''{0}'' is no longer supported. Use {1} or later.
 # more programming problems:
```

Comment:
Not sure, as for the most users the question will be "what the 'source level' is" and "how I can fix that level" :-)

Reply:
The proposed new message is indeed quite a broad statement :) 

another attempt:
`Compiling for Java version ''{0}'' is no longer supported. Minimal supported version is ''{1}''`

problem is: the message cannot say use `-source 1.8` or similar, because the error is a catch-all for source, target & compliance.

Reply:
> Not sure, as for the most users the question will be "what the 'source level' is" and "how I can fix that level" :-)

Ah. I meant source "level" as in source "form"! I like Stephan's suggestion better than mine which is ambiguous and overloads "level" in confusing way.